contracts/strategies/mstable/IdleMStableStrategy.sol

```diff
@@ -119,15 +120,14 @@ contract IdleMStableStrategy is Initializable, OwnableUpgradeable, ERC20Upgradea
         lastIndexedTime = block.timestamp;
         //------//-------//
 
-        (, , rewardLastRound) = vault.unclaimedRewards(address(this));
         transferOwnership(_owner);
         IERC20Detailed(_underlyingToken).approve(_strategyToken, type(uint256).max);
         ISavingsContractV2(_strategyToken).approve(_vault, type(uint256).max);
     }
 
     /// @notice redeem the rewards. Claims all possible rewards
     /// @return rewards amount of underlyings (mUSD) received after selling rewards
-    function redeemRewards() external onlyIdleCDO returns (uint256[] memory rewards) {
+    function redeemRewards() external onlyOwner returns (uint256[] memory rewards) {
```

Comment:
why `onlyOwner`?

Reply:
Because the IdleCDO can only call `redeemRewards(bytes)` and ideally the `redeemRewards()` should be a backup method called externally